bsms.mediawiki

```diff
@@ -280,74 +280,104 @@ wsh(sortedmulti(2,[4b6a9b93/48'/1'/1']tpubDDpNnt2fg3G4ukE8gsifBKank5sGemyMDa6gUj
-** IV (hex): 730668fdec78b59be985d6fd0b1c0009
-** CIPHERTEXT (hex): 0995596112867152f75ae0376cfd7408b96d7ee20e21ef4f9278407cae6ddfe3c3f005e4ff0ddb541f63fae03962e1406c6d0d197417aab1df096eeb4d29975622a92da149bcdcf5b5c7c91049d026a905e3942683ee97f44635ffd224e29342a1ee0e7fbb8c85862cf3f63462ab279c165644d1349e51987e1ea8778c471cb95e27ee79d9fbe01ae0bbf9740c9b3a23b70217153dbddda8ba8e0c49423a59c6b38b57e0d2c4d7fd74a6e110c49dca102369025fd009215e2db2e79fb555b77d44d24787dc75b0a6e71a0b01ef74ad5b7496a13b77ebe9335973b4cc668293dd1f72866c742976cf8d814be5f005d43a1722ae0a6ea30db2d2f4ba144d502fa236e1c76e23310163ec7787
-** <tt>signer_2_key.dat</tt>: <pre>730668fdec78b59be985d6fd0b1c000943e0ca496e386fedca46a9531f6c64ee0995596112867152f75ae0376cfd7408b96d7ee20e21ef4f9278407cae6ddfe3c3f005e4ff0ddb541f63fae03962e1406c6d0d197417aab1df096eeb4d29975622a92da149bcdcf5b5c7c91049d026a905e3942683ee97f44635ffd224e29342a1ee0e7fbb8c85862cf3f63462ab279c165644d1349e51987e1ea8778c471cb95e27ee79d9fbe01ae0bbf9740c9b3a23b70217153dbddda8ba8e0c49423a59c6b38b57e0d2c4d7fd74a6e110c49dca102369025fd009215e2db2e79fb555b77d44d24787dc75b0a6e71a0b01ef74ad5b7496a13b77ebe9335973b4cc668293dd1f72866c742976cf8d814be5f005d43a1722ae0a6ea30db2d2f4ba144d502fa236e1c76e23310163ec7787</pre>
+** HMAC_KEY (hex): 7f34b0be728f1b67b305eab3c7f40bc2200ac798a280539d5a38dacb13e2d333
+** MAC (hex): 8cb18a39e1a3c2e1ae4e4355d535d1aaf0313cece0ff9d8940948d5a0bacb443
+** IV (hex): 8cb18a39e1a3c2e1ae4e4355d535d1aa
+** CIPHERTEXT (hex): 87fbb831729885c03f507b737df5c123346f25d850608249ac1c30828f489feaf868b6cdd779b1c2db6e1f26bf646c6cde64fd4a2de99548b47a43a890a78802432ce806eef01be4ede16e2ffbb582dc73af64e4fceca0f23b9b0a8aaff2bbb82d3cc068c6342d41f3b7b3a2fed648e7b22dfc89302faf9ad1b7051e766ae53ec62007ba4f2651927cec87828dcbc3c8c3cb7997ff941de6ffdb232b5e0f3c52b2b28aceaf50eae4bc179c433b738df0779206cfc34df0d61475e81ae8b5da8ae15ebbe322e029ae6e66449a4e5cb52212194a5c42f14350348eb46a491dc1cdd646bf9e1152a4f7a4b3e06a5b6dd77157fe6864ca63bc7fccec34fa4854322729c8c7b54a76b1041b73be
+** <tt>signer_2_key.dat</tt>: <pre>8cb18a39e1a3c2e1ae4e4355d535d1aaf0313cece0ff9d8940948d5a0bacb44387fbb831729885c03f507b737df5c123346f25d850608249ac1c30828f489feaf868b6cdd779b1c2db6e1f26bf646c6cde64fd4a2de99548b47a43a890a78802432ce806eef01be4ede16e2ffbb582dc73af64e4fceca0f23b9b0a8aaff2bbb82d3cc068c6342d41f3b7b3a2fed648e7b22dfc89302faf9ad1b7051e766ae53ec62007ba4f2651927cec87828dcbc3c8c3cb7997ff941de6ffdb232b5e0f3c52b2b28aceaf50eae4bc179c433b738df0779206cfc34df0d61475e81ae8b5da8ae15ebbe322e029ae6e66449a4e5cb52212194a5c42f14350348eb46a491dc1cdd646bf9e1152a4f7a4b3e06a5b6dd77157fe6864ca63bc7fccec34fa4854322729c8c7b54a76b1041b73be</pre>
+
+* Signer 3
+** MASTER_KEY_FINGERPRINT: 59b482bc
+** PRIVATE_KEY (m/48’/1’/1’): L3EiGK6P2aDnCe6g2c1Pb9odV5oNsvy8irNjkTSGtLPMvn18jMUK
+** XPUB (m/48’/1’/1’): xpub6CkFnjTzPGuDkwNBeQN1DsftwmcPsaqAxp3nLBM3BzJ9BoXwZiAeniH2J4m67U4Vm62SnF6hLXVjdmSa7W5aW2n9MvDetEPsqVnew5Lrg7L
+** Legacy signature
+** <tt>signer_2_key.bsms</tt>:
```

Comment:
signer_3_key.bsms

Reply:
fixed in https://github.com/nunchuk-io/bips/pull/1/commits/b267b20b661b8055233363707feec90f9b6edebe